readme.md

```diff
@@ -20,8 +20,8 @@ Create a pages with product card using `flexbox`, `BEM` and `SCSS` based on [thi
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://Leh-02.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://Leh-02.github.io/layout_product-cards/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="754" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/d1221af0-4110-47e4-bda7-7f49df8a9fc8">